internal/component/beyla/ebpf/beyla_linux.go

```diff
@@ -337,14 +448,19 @@ func (a *Arguments) Convert() (*beyla.Config, error) {
 	}
 	cfg.Prometheus = a.Metrics.Convert()
 	cfg.NetworkFlows = a.Metrics.Network.Convert()
+	if a.EnforceSysCaps {
+		cfg.EnforceSysCaps = true
+	}
+	cfg.EBPF = a.EBPF.Convert()
+	cfg.Filters = a.Filters.Convert()
 
 	if a.Debug {
 		// TODO: integrate Beyla internal logging with Alloy global logging
 		lvl := slog.LevelVar{}
 		lvl.Set(slog.LevelDebug)
 		cfg.ExternalLogger(slog.New(slog.NewTextHandler(os.Stdout, &slog.HandlerOptions{
 			Level: &lvl,
-		})).Handler(), a.Debug)
+		})).Handler(), false)
```

Comment:
Why was this changed?

Reply:
i was testing something, thanks for catching it